### PR TITLE
Lego 3506 global alert

### PR DESCRIPTION
**Pull request by majabnils**
# PR Checklist
https://elvia.atlassian.net/wiki/spaces/TEAMATOM/pages/10427498683/Review+prosess

- [ ] Bumpet package?
- [ ] Updated changelog?
- [ ] Correct date in changelog?

## Describe PR briefly:
https://elvia.atlassian.net/browse/LEGO-3506

### CODE REVIEW

**Comment by TomSchrier** [comment]
The documentation needs to be updated. The current paragraph tells the user to put the Global Alert at the top of their app, however, the new design guidelines recommend putting it at the bottom.

Maybe something like this?

```
Global Alert
Global alert is in most cases initiated by the application or system, but it can also be used when an important message needs to be flagged. It is a general message to all users and is not associated with user actions. The global alert appears on all pages of the application/website and extends in full length at the bottom of the viewport. It can include links or actions such as buttons and a close button.

NB: Remember to fix the global alert to the bottom of your application and ensure that it extends the full length of the page! 
```

File: packages/elvis/src/components/alert.scss [hunk 1]
```diff
@@ -102,126 +103,29 @@ $global-mobile-notitle-padding: 14px; // -2px border
...
-        width: 100%;
-        max-height: 0;
-        margin-top: 0;
-        opacity: 0;
-        pointer-events: none;
-        transition:
-          max-height 0.18s cubic-bezier(0, 0.18, 0.9, 1),
-          opacity 0.17s cubic-bezier(0.4, 0.01, 0.78, 1.1),
-          margin-top 0.17s cubic-bezier(0, 0.18, 1, 1);
+        color: var(--e-color-text-5);
```
[the remark this answers is not in the document]
Comment (majabnils) [reply packages/elvis/src/components/alert.scss]: You're right! Didn't notice that 👀

File: packages/elvis/src/components/alert.scss [hunk 2]
```diff
@@ -76,7 +76,8 @@ $global-mobile-notitle-padding: 14px; // -2px border
     width: 100%;
     max-width: unset;
     padding: $global-padding;
-    border-color: var(--e-color-signal-info);
+    border: none;
+    background-color: var(--e-color-background-element-5);
     @include breakpoints.breakpoint-up(Mobile) {
       padding: $global-mobile-padding;
     }
```
Comment (TomSchrier) [inline packages/elvis/src/components/alert.scss:83]: Now that `border` is set to `none`, please ensure the padding is still correct on both desktop and mobile. I noticed that the variable `$global-mobile-padding` has a comment "`// -2px border`" (see line 10). Should this padding be updated?

File: packages/elvis-colors/gulpfile.ts [hunk 3]
```diff
@@ -29,10 +29,10 @@ const generateElvisColorsCss = async () => {
   fileContent += `:root {\n`;
   Object.entries(rootVariables).forEach(([name, color]) => (fileContent += `\t${name}: ${color};\n`));
   fileContent += `}\n`;
-  fileContent += `.e-theme-light,\n:root {\n`;
+  fileContent += `.e-theme-light, .e-alert--global.e-alert--warn *, .e-theme-dark .e-alert--global:not(.e-alert--warn):not(.e-alert--error) *, \n:root {\n`;
   Object.entries(lightVariables).forEach(([name, color]) => (fileContent += `\t${name}: ${color};\n`));
   fileContent += `}\n`;
-  fileContent += `.e-theme-dark, .e-color-background-3, [class*='e-'][class*="--inverted"] {\n`;
+  fileContent += `.e-theme-dark, .e-color-background-3, .e-theme-light .e-alert--global:not(.e-alert--warn):not(.e-alert--error) *, .e-alert--global.e-alert--error *, [class*='e-'][class*="--inverted"] {\n`;
```
Comment (TomSchrier) [inline packages/elvis-colors/gulpfile.ts:35]: What does this do?

Comment (majabnils) [reply packages/elvis-colors/gulpfile.ts:35]: It turns everything inside to light theme or dark theme based on if it is warning, informative or error.

File: packages/elvis/src/components/alert.scss [hunk 4]
```diff
@@ -102,126 +103,29 @@ $global-mobile-notitle-padding: 14px; // -2px border
 
     .e-alert__content .e-alert__text {
       max-width: unset;
+      color: var(--e-color-text-1);
     }
```
[the remark this answers is not in the document]
Comment (majabnils) [reply packages/elvis/src/components/alert.scss:107]: Hmm maybe percy ignores the gulpfile update??

[the remark this answers is not in the document]
Comment (majabnils) [reply packages/elvis/src/components/alert.scss:107]: Or maybe we just don't use e-theme-light in light theme for the percy tests? I use the e-theme-light selector to set the theme to dark on the informative alert.

[the remark this answers is not in the document]
Comment (majabnils) [reply packages/elvis/src/components/alert.scss:107]: The percy tests seem to work now as I removed the selector for e-theme-light!

**Review by TomSchrier — APPROVED** [review]
LGTM

**Pull request merged**